pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -191,6 +194,11 @@ async def run(self):
         _LOGGER.info("Updating ocp-build-data...")
         await self.update_build_data(advisories, jira_issue_key)
 
+        if "advance" in advisories.keys():
+            self.advance_release = True
```

Comment:
I would one more check here, which is "advance advisory is alive and in an editable state"
I can see a scenario where we run this after advance advisory has shipped, just to double check everything

Reply:
Added in https://github.com/openshift-eng/art-tools/pull/341/commits/c8de8dcfa1d6e9331e182b3d165de25ce6ff4dd9

hope `raise VerificationError` will stop the code from proceeding.